frontend/src/components/parties/PartyProfile.js

```diff
@@ -103,6 +106,20 @@ export class PartyProfile extends Component {
     });
   };
 
+  deleteParty = () => {
+    const { id } = this.props.match.params;
+    this.props.deleteParty(id).then(() => {
+      const page = String.DEFAULT_PAGE;
+      const per_page = String.DEFAULT_PER_PAGE;
+      this.props.history.push(
+        router.CONTACT_HOME_PAGE.dynamicRoute({
+          page,
+          per_page,
```

Comment:
Small thing, but the assignment on lines 112-113 is a bit unnecessary. May as well just pass in the values here:
```javascript
router.CONTACT_HOME_PAGE.dynamicRoute({
  page: String.DEFAULT_PAGE,
  per_page: String.DEFAULT_PER_PAGE,
})
```

Reply:
fixed any way